src/OrchardCore.Modules/OrchardCore.OpenId/Validators/OpenIdUrlValidator.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace OrchardCore.OpenId.Validators
+{
+    public class OpenIdUrlValidator
+    {
+        public IEnumerable<ValidationResult> ValidateUrls(string memberName, string member)
+        {
+            var attribute = new UrlAttribute();
+            foreach (var url in member.Split(' '))
```

Comment:
I usually use both `{ ' ', ','}` to split items, as long as `,` is not valid in an entry, and also use `StringSpitOptions.RemoveEmptyEntries` in case there two spaces are typed.

Reply:
Ultimately, we'll probably want to get rid of `UrlAttribute` as it's overly aggressive (it basically only accepts the http/https/ftp schemes, which doesn't work with custom schemes like `ms-app`)